includes/routes/event/edit.php

```diff
@@ -39,20 +40,18 @@ public function handle( int $event_id ): void {
 		}
 
 		include ABSPATH . 'wp-admin/includes/post.php';
-		$event_page_title              = 'Edit Event';
-		$event_form_name               = 'edit_event';
-		$css_show_url                  = '';
-		$event_title                   = $event->title();
-		$event_description             = $event->description();
-		$event_status                  = $event->status();
-		list( $permalink, $post_name ) = get_sample_permalink( $event->id() );
-		$permalink                     = str_replace( '%pagename%', $post_name, $permalink );
-		$event_url                     = get_site_url() . gp_url( wp_make_link_relative( $permalink ) );
-		$event_timezone                = $event->timezone();
-		$event_start                   = $event->start();
-		$event_end                     = $event->end();
-		$create_delete_button          = false;
-		$visibility_delete_button      = 'inline-flex';
+		$event_page_title         = 'Edit Event';
+		$event_form_name          = 'edit_event';
+		$css_show_url             = '';
+		$event_title              = $event->title();
+		$event_description        = $event->description();
+		$event_status             = $event->status();
+		$event_url                = Urls::event_details_absolute( $event_id );
```

Comment:
What do you think about introducing an alias `$event->url()`?

Reply:
I'm not sure about this one because there's `Urls::event_details_absolute()` and `Urls::event_details()`. So we'd need to choose one to use in the `$event->url()` method.

I think then it could become confusing because you'd need to remember whether `$event->url()` returns the absolute or relative URL.

Reply:
Or maybe all functions in `Urls` should return the absolute URL?

Reply:
I think just in this context it's preferrable to avoid id juggling but maybe it's one-off enough to be ok? Don't have a strong opinion.